scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -918,6 +924,428 @@ cdef class TransformedDensityRejection(Method):
+        * ``cdf``: CDF of the distribution. This method is optional. If provided, it
+          enables the calculation of "u-error". See `u_error`. Must have the same
+          signature as the PDF.
+
+    mode : float, optional
+        (Exact) Mode of the distribution. Default is ``None``.
+    center : float, optional
+        Approximate location of the mode or the mean of the distribution. This location
+        provides some information about the main part of the PDF and is used to avoid
+        numerical problems. Default is ``None``.
+    domain : list or tuple of length 2, optional
+        The support of the distribution. Must not contain `nan` values.
+        Default is ``None``. When ``None``:
+
+        * If a ``support`` method is provided by the distribution object
+          `dist`, it is used to set the domain of the distribution.
+        * Otherwise the support is assumed to be :math:`(-\infty, \infty)`.
+
+    order : int, optional
+        Order of the interpolating polynomial. Valid orders are between 3 and 17.
+        Higher orders result in fewer intervals for the approximations. Default
+        is 3.
+    u_resolution : float, optional
+        Set maximal tolerated u-error. Values of u_resolution must at least 1.e-15 and
+        1.e-5 at most. Notice that the resolution of most uniform random number sources
+        is 2-32= 2.3e-10. Thus a value of 1.e-10 leads to an inversion algorithm that
+        could be called exact. For most simulations slightly bigger values for the
+        maximal error are enough as well. Default is 1e-10.
+    max_intervals : int, optional
+        Set maximum number of intervals. max_ivs must be at least 100 and at most 1000000.
+        Default is 10000.
+    keep_cdf : bool, optional
+        If the PDF is given, then the CDF is computed numerically from the given PDF using
+        adaptive Gauss-Lobatto integration. Thus a table of CDF points is stored to keep
+        the number of evaluations of the PDF minimal. Usually this table is discarded when
+        the setup is completed. If ``keep_cdf`` is TRUE, then this table is kept and the
+        approximated CDF can be evaluated by calling the `cdf` method. Default is ``False``.
+    random_state : {None, int, `numpy.random.Generator`,
+                        `numpy.random.RandomState`}, optional
+
+        A NumPy random number generator or seed for the underlying NumPy random
+        number generator used to generate the stream of uniform random numbers.
+        If `random_state` is None (or `np.random`), the `numpy.random.RandomState`
+        singleton is used.
+        If `random_state` is an int, a new ``RandomState`` instance is used,
+        seeded with `random_state`.
+        If `random_state` is already a ``Generator`` or ``RandomState`` instance then
+        that instance is used.
+
+    References
+    ----------
+    .. [1] G. Derflinger, W. Hörmann, and J. Leydold (2008). Numerical inversion when
```

Comment:
Please use the following citation: ... ACM Transactions on Modeling and Computer Simulation (TOMACS) 20.4 (2010): 1-25.

Reply:
Done.